tests/explainable/test_handlers_components.py

```diff
@@ -376,49 +376,64 @@ def model_ce():
 def test_consequent_eq_neq(plate_size, event_shape):
```

Comment:
All the logic about `event_shape` and `event_dim` in this test needs to be restored, see other comments.

Reply:
The logic for `event_shape` has been restored.